### PR TITLE
topology-manager: promoto TopologyManagerPolicyOptions to stable

### PR DESCRIPTION
**Pull request by PiotrProkop**
### Description

Promote TopologyManagerPolicyOptions to stable.

### Issue

Related-to: https://github.com/kubernetes/enhancements/issues/3545

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/feature-gates/topology-manager-policy-options.md [hunk 1]
```diff
@@ -6,13 +6,16 @@ _build:
...
 
 stages:
-  - stage: alpha 
+  - stage: alpha
     defaultValue: false
     fromVersion: "1.26"
     toVersion: "1.27"
   - stage: beta
     defaultValue: true
     fromVersion: "1.28"
```
Comment (tengqm) [inline content/en/docs/reference/command-line-tools-reference/feature-gates/topology-manager-policy-options.md:15]: `toVersion: "1.31"`

File: content/en/docs/tasks/administer-cluster/topology-manager.md [hunk 2]
```diff
@@ -223,12 +223,11 @@ You can toggle groups of options on and off based upon their maturity level usin
 
 You will still have to enable each option using the `TopologyManagerPolicyOptions` kubelet option.
 
-### `prefer-closest-numa-nodes` (beta) {#policy-option-prefer-closest-numa-nodes}
+### `prefer-closest-numa-nodes` (stable) {#policy-option-prefer-closest-numa-nodes}
```
Comment (tengqm) [inline content/en/docs/tasks/administer-cluster/topology-manager.md:226]: Adding "(stable)" is optional.
For most, if not all, features that are stable, we don't explicitly mention it.

File: content/en/docs/tasks/administer-cluster/topology-manager.md [hunk 3]
```diff
@@ -223,12 +223,11 @@ You can toggle groups of options on and off based upon their maturity level usin
...
 
-The `prefer-closest-numa-nodes` option is beta since Kubernetes 1.28. In Kubernetes {{< skew currentVersion >}}
-this policy option is visible by default provided that the `TopologyManagerPolicyOptions` and
-`TopologyManagerPolicyBetaOptions` [feature gates](/docs/reference/command-line-tools-reference/feature-gates/)
-are enabled.
+The `prefer-closest-numa-nodes` option is GA since Kubernetes 1.32. In Kubernetes {{< skew currentVersion >}}
+this policy option is visible by default provided that the `TopologyManagerPolicyOptions`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled.
```
Comment (tengqm) [inline content/en/docs/tasks/administer-cluster/topology-manager.md:230]: Em ... isn't this feature "locked-to-true"?
In other words, is it possible to disable it after GA?

Comment (PiotrProkop) [reply content/en/docs/tasks/administer-cluster/topology-manager.md:230]: Ha this one is kinda complex. So for GA the behavior stays the same as for beta. The `prerfer-closest-numa-nodes-nodes` is disabled by default and has to be enabled by setting `prefer-closest-numa-nodes=true` in kubelet config. But if you disable `TopologyManagerPolicyOptions` feature gate then you won't be even able to set this parameter. This feature follows the same pattern as `CPUManagerPolicyOptions` which are already stable 😄  

So answering your question:
- no this feature is not locked-to-true, by default it's not enabled
- yes, you can disable/enable it by passing special params to kubelet

File: content/en/docs/tasks/administer-cluster/topology-manager.md (hunk 2 above)
Comment (PiotrProkop) [reply content/en/docs/tasks/administer-cluster/topology-manager.md:226]: 👍 will change that

File: content/en/docs/reference/command-line-tools-reference/feature-gates/topology-manager-policy-options.md (hunk 1 above)
Comment (PiotrProkop) [reply content/en/docs/reference/command-line-tools-reference/feature-gates/topology-manager-policy-options.md:15]: 👍

**Review by dipesh-rawat — COMMENTED** [review]
Upstream PR: https://github.com/kubernetes/kubernetes/pull/128124
Status: merged

/approve

**Pull request merged**